<launch>

    <node pkg="auv_vision" name="gate_locator" type="gate_locator" output="screen">
        <remap from="/cam_front_1/image_raw" to="/ROV_model_URDF/camera1/image_raw"/>
    </node>

    <node pkg="auv_pilot" name="simple_simulation_fsm" type="simple_simulation_fsm.py" output="screen" />

    <node pkg="auv_pilot" name="simple_simulation_pilot" type="simple_simulation_pilot" output="screen" />

</launch>